conda.recipe/meta.yaml

```diff
@@ -41,46 +41,6 @@ requirements:
     - {{ dep.lower() }}
     {% endfor %}
 
-test:
```

Comment:
we may want to leave the top-level `test` stanza but delete everything under it except the `commands` and `imports` sub-stanzas. IIRC conda-build will use those to perform  very rudimentary smoke-tests on the built package.